### PR TITLE
reco.cc: add InclusiveKinematicseSigma backwards compatibility for now

### PR DESCRIPTION
**Pull request by veprbl**
Provide a graceful deprecation for https://github.com/eic/EICrecon/pull/1572

### CODE REVIEW

File: src/global/reco/reco.cc [hunk 1]
```diff
@@ -171,6 +171,18 @@ void InitPlugin(JApplication *app) {
         app
     ));
 
+    // InclusiveKinematicseSigma is deprecated and will be removed, use InclusiveKinematicsESigma instead
+    app->Add(new JOmniFactoryGeneratorT<CollectionCollector_factory<edm4eic::HadronicFinalState>>(
```
Comment (wdconinc) [inline src/global/reco/reco.cc:175]:
```suggestion
    app->Add(new JOmniFactoryGeneratorT<CollectionCollector_factory<edm4eic::InclusiveKinematics>>(
```

**Comment by wdconinc** [comment]
What is this? Windows?

**Review by wdconinc — DISMISSED** [review]

**Comment by veprbl** [comment]
> What is this? Windows?

Dot files are not converting, there might be a regex issue.

**Comment by wdconinc** [comment]
Offending dot file has a subgraph cluster_18 that's just empty... But I think what really fixes this for me is upgrading graphviz. I'll see to push that out to nightly.

**Comment by wdconinc** [comment]
> Offending dot file has a subgraph cluster_18 that's just empty....

Must have always been empty due to LFHCAL duplicate naming:
https://github.com/eic/EICrecon/blob/4a2e46e3b688c9818cea6a1c4eb247312d176cfc/.github/janadot.groups#L14

**Comment by veprbl** [comment]
No cigar

**Comment by wdconinc** [comment]
> No cigar

Yeah, I'd restarted some jobs which should have picked up a new graphviz, and no luck.

Maybe we need to make the graphviz job to allow failure. And figure out the issue on the side.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**